src/componets/Card/Card.tsx

```diff
@@ -0,0 +1,64 @@
+import { Card, Rate } from 'antd';
+import styles from './card.module.css';
+import { data } from './constants';
+import { useState } from 'react';
+import useProductCardHandlers from '../../hooks/useProductCardHandlers';
+
+const ProductCard = () => {
+  const {
+    isHovered,
+    isClicked,
+    isLiked,
+    isMouseDown,
+    handleClickBuy,
+    handleMouseDown,
+    handleMouseUp,
+    handleMouseLeave,
+    handleMouseEnter,
+    handleClickLike,
+  } = useProductCardHandlers();
+
+  return (
+    <>
+      {data.map((item) => (
+        <Card className={styles.cardConatiner} key={item.id}>
+          <div className={styles.cardContainerTop}>
+            <img src={item.img} className={styles.productImage} />
+            <div className={styles.cardConatinerLike}>
+              <img
+                onClick={handleClickLike}
+                src={isLiked ? item.likeIcon : item.likeIconClick}
+                alt="Like"
+                className={styles.likeIcon}
+              />
+              <img src={item.colorPalette} className={styles.colorPalette} />
```

Comment:
you should write an alt attribute for all img tags

Reply:
Added)